openshift_metrics/utils.py

```diff
@@ -290,53 +290,61 @@ def write_metrics_by_namespace(
         metrics["SU_CPU_HOURS"] += su_count_hours
 
         row = [
+            report_month,
             namespace,
             metrics["pi"],
-            report_start_date,
-            report_end_date,
-            str(metrics["_cpu_hours"]),
-            str(metrics["_memory_hours"]),
-            SU_CPU,
+            "", #Invoice Address
+            "", #Institution
+            "", #Institution - Specific Code
             str(metrics["SU_CPU_HOURS"]),
+            SU_CPU,
+            "", #Rate
+            "" #Cost
         ]
         rows.append(row)
 
         if metrics["SU_A100_GPU_HOURS"] != 0:
             row = [
+                report_month,
                 namespace,
                 metrics["pi"],
-                report_start_date,
-                report_end_date,
-                "NA",
-                "NA",
-                SU_A100_GPU,
+                "", #Invoice Address
+                "", #Institution
+                "", #Institution - Specific Code
                 str(metrics["SU_A100_GPU_HOURS"]),
+                SU_A100_GPU,
+                "", #Rate
+                "" #Cost
             ]
             rows.append(row)
 
         if metrics["SU_A10_GPU_HOURS"] != 0:
             row = [
+                report_month,
                 namespace,
                 metrics["pi"],
-                report_start_date,
-                report_end_date,
-                "NA",
-                "NA",
-                SU_A10_GPU,
+                "", #Invoice Address
+                "", #Institution
+                "", #Institution - Specific Code
                 str(metrics["SU_A10_GPU_HOURS"]),
+                SU_A10_GPU,
```

Comment:
technically this is A2 and there is a V100 that we will eventually need to add as well.